plugin/src/main/scala/com/typesafe/genjavadoc/Output.scala

```diff
@@ -106,10 +107,16 @@ trait Output { this: TransformCake ⇒
     val methods = cls.members collect {
       case m: MethodInfo ⇒
         if (m.ret.endsWith("$") && classes.exists(_.name == m.name))
-          m.copy(comment = Seq("/**", " * Accessor for nested Scala object", " */"))
+          m.copy(comment = Seq("/**", " * Accessor for nested Scala object", " * @return (undocumented)", " */"))
```

Comment:
I would not do this if it is not an error. That is just stupid, and sprinkes the documentation with unprofessional comments.